dev-itpro/performance/performance-developer.md

```diff
@@ -64,10 +64,18 @@ Things that have historically caused performance on pages that are exposed as en
 - Many SIFT fields 
 - FactBoxes 
  
+Avoid exposing calculated fields, because calculated fields are expensive. Try to move them to a separate page or to refactor the code so the value is stored on the physical table (if applicable). Complex types are also a performance hit because they take a lot of time to calculate. 
+
+Don't use temp tables as a source if you have a lot of records. Temp tables based APIs are a performance hit. The server has to fetch and insert every record, and there is no caching on data in temp tables. Paging becomes difficult to do in a performant manner. A rule of thumb is if you have more than 100, 200 records do not use temp tables.
```

Comment:
Hi Kennie, can you clarify this statement:
A rule of thumb is if you have more than **100, 200** records do not use temp tables.

Should this be either:
A rule of thumb is if you have more than 100 records, do not use temp tables.

or 
A rule of thumb is if you have more than 200 records, do not use temp tables.

Reply:
Let go with 100 records

Reply:
Thanks!